File-Formats/DocIO/Working-with-Find-and-Replace.md

```diff
@@ -1019,6 +1019,116 @@ document.Close();
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Find-and-Replace/Find-and-replace-text-with-image).
 
+## Find next and replace with formatted text
+
+The following code example illustrates how to find next and replace with formatted text in a Word document:
```

Comment:
@sharonaumao - Please change the descriptions as followed "The following code example illustrates how to replace the text extended to several paragraphs with particular text in the document."

Reply:
@Ramaraj-Marimuthu -I have changed